### DISCUSSION TITLE
Documentation using Vite

### DISCUSSION THREAD

**Discussion by masrodjie** [vanilophp/framework#131] [General]
Laravel 9 using vite.
Is there any documentation to use vite?

**Comment by fulopattila122** [comment]
> Is there any documentation to use vite?

Yes: https://laravel.com/docs/9.x/vite :stuck_out_tongue_winking_eye: 

The Vanilo Framework doesn't contain any frontend code, it's a backend-only library.

In case you're referring to the Vanilo Admin package, see https://github.com/vanilophp/admin/issues/2

**Comment by dev-adeola** [comment]
@fulopattila122 
Hi,

I am very thankful for the great work you are doing on vanilo ecommerce.
Please, Some functions seem not to work. 
I couldn't load the image properly
and
category assignment to products gives an error.
***Route [vanilo.product.show] not defined.***

Please, I would love your kind help and insight to what I need to do.

***I am working in a localhost | wampserver***

____________________________________________________

***Route [vanilo.product.show] not defined.***
***Call to undefined method App\Models\User::customersVisible()***

**Reply by fulopattila122** [reply]
@dev-adeola Make sure you completed all the steps in the Admin Installation guide: https://vanilo.io/docs/3.x/admin-installation

**Comment by fulopattila122** [comment]
Good news for anyone willing to write the Vite Docs for Vanilo Admin: https://github.com/vanilophp/docs/issues/20
Now, it's a hacktoberfest issue :raised_hands: Go for the :shirt: or for the :deciduous_tree:
